src/main/resources/containers.json

```diff
@@ -1250,8 +1250,6 @@
     ]
   },
   {
-    "artifact_id": "glassfish",
-    "group_id": "org.glassfish.main.distributions",
     "name": "Arquillian Container GlassFish Managed",
```

Comment:
In theory we could have a way to query container adapter code for this information instead of maintaining it in the YAML (which already is a duplication). What if that changes? We won't know about it unless we did the change or someone told us about it :)